### PR TITLE
Swag????????????

### PR DESCRIPTION
**Pull request by BrennanB**
WEFqergqerg @kmbibic

### CODE REVIEW

**Comment by kmbibic** [comment]
This looks cool

File: Yo.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print "yo"
```
Comment (kmbibic) [inline Yo.py:1]: Consider adding a `"Hi"` as well

Comment (BrennanB) [inline Yo.py:1]: Added

**Comment by mrael2** [comment]
Testing, testing, 123

**Pull request closed without merging**